queries/vim/highlights.scm

```diff
@@ -0,0 +1,91 @@
+;; Keywords
+
+[
+  "if"
+  "else"
+  "elseif"
+  "endif"
+] @conditional
+
+[
+  "try"
+  "catch"
+  "finally"
+  "endtry"
+  "throw"
+] @exception
+
+[
+  "for"
+  "endfor"
+  "in"
+  "while"
+  "endwhile"
+] @repeat
+
+[
+  "function"
+  "endfunction"
+] @keyword.function
+
+[ (bang) (spread) ] @punctuation.special
+
+[ (no_option) (inv_option) (default_option) (option_name) ] @constant.builtin
```

Comment:
are options constant when you can set them? `@variable.builtin`?

Reply:
Huh, yup, that seems better.